### PR TITLE
Fix NetworkFailureToast blocking pointer event even it's invisible #7

### PR DESCRIPTION
**Pull request by wallacesky**
Unmount NetworkFailureToast after animation completed

### CODE REVIEW

File: src/FadeAnimation.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ interface Props {
   children: React.ReactNode;
   duration: number;
   style?: StyleProp<ViewStyle>;
+  onFinishedAnimate?: () => void;
```
Comment (louischan-oursky) [inline src/FadeAnimation.tsx]: onAnimationEnd seems a better name.

File: src/NetworkFailureToast.tsx [hunk 2]
```diff
@@ -85,9 +87,30 @@ export default class NetworkFailureToast extends React.PureComponent<
...
+      this.setState(
+        {
+          isHidden: false,
+        },
+        () => {
+          this.setState({
+            isNetworkConnected: isConnected,
+          });
+        }
+      );
```
Comment (cychiuae) [inline src/NetworkFailureToast.tsx]: Please add comment to document when separate setState instead of setting all state at one time

**Comment by cychiuae** [comment]
@wallacesky Ping for this

**Comment by wallacesky** [comment]
@cychiuae @louischan-oursky I updated it accordingly

**Pull request merged**